_includes/demo.html

```diff
@@ -32,7 +32,7 @@
       document.getElementById('progress').style.display = 'none';
     });
 
-    wavesurfer.load('/example/media/small-demo.wav');
+    wavesurfer.load('/example/media/demo.wav');
```

Comment:
I've uploaded a `small-demo.wav`, too.

Reply:
ok ill revert this change then.

Reply:
can't resolve this afterall since the `examples/media/` directory where you placed the file is cleaned and synced with the master branch after each release.